addons/website_portal_purchase/security/ir.model.access.csv

```diff
@@ -0,0 +1,12 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_purchase_order,purchase.order,purchase.model_purchase_order,base.group_portal,1,0,0,0
+access_purchase_order_line,purchase.order.line,purchase.model_purchase_order_line,base.group_portal,1,0,0,0
+access_product_list,product.pricelist,product.model_product_pricelist,base.group_portal,1,0,0,0
```

Comment:
id incoherent with model

Is it necessary to give access to pricelists to portal users ? Seems louche.